packages/cpp/src/browser/cpp-preferences.ts

```diff
@@ -61,6 +61,16 @@ export const cppPreferencesSchema: PreferenceSchema = {
             description: 'Specify the arguments to pass to clangd when starting the language server.',
             default: '',
             type: 'string'
+        },
+        'cpp.clang-tidy': {
+            description: 'is a clang-based C++ “linter” tool.',
```

Comment:
Since the `preference` here is used to determine if `clang-tidy` should be used by `clangd`, a more appropriate description may be:

```json
description: 'Enable/disable C/C++ linting.`
```

or 

```json
description: 'Control whether clang-tidy is enabled for C/C++ files or not.'
```

?